.browserslistrc

```diff
@@ -1,14 +0,0 @@
-# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
```

Comment:
What was the reason for removing this file?

Reply:
I'm actually not sure, I don't recall going in and deleting this file. Is it possible that it was automatically deleted when I ran ng update?

Reply:
After some research I couldn't find a good reason why this file may have been deleted, so I added it back in and ran `npx browserslist@latest --update-db` to make sure everything was up to date.